students/zach_thomson/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,14 @@
+def fizzBuzz():
```

Comment:
pep8 style violation: use snake_case style for var names and func names -> `fizz_buzz`

Reply:
Ok, good to know. I will use that next time.